### PR TITLE
[Pages] Add warning to prod branch control

### PR DESCRIPTION
**Pull request by WalshyDev**
A long-standing issue and people continue to get confused by this page so let's add a warning with an action item for users.

### CODE REVIEW

**Comment by Erisa** [comment]
Looks like the same goal as the warning i added in #13180 - was this a better place to put it or is there value in having both?

The placement of mine was as recommended in https://github.com/cloudflare/cloudflare-docs/pull/13180#pullrequestreview-1923117794

If we do need both, Kate's feedback on the curl placeholders would apply here too: https://github.com/cloudflare/cloudflare-docs/pull/13180/files/43831d91616de4d9b2a7173384f450792ae85088#diff-261851dd6ea58e2a549870365f622562ae5e64b66f57cbb3d1cc567423971469

**Comment by WalshyDev** [comment]
This is the page that keeps confusing people so I definitely think we need _something_ here

**Review by deadlypants1973 — APPROVED** [review]

File: content/pages/configuration/branch-build-controls.md [hunk 1]
```diff
@@ -3,37 +3,50 @@ pcx_content_type: concept
...
+## Production branch control
 
-To configure deployment options, go to your Pages project > **Settings** > **Builds & deployments** > **Configure Production deployments**. Pages will default to setting your production environment to the branch you first push, but you can set your production to another branch if you choose. 
+{{<Aside type="warning" header="Direct Upload">}}
 
-You can also enable or disable automatic deployment behavior on the production branch by checking the **Enable automatic production branch deployments** box. You must save your settings in order for the new production branch controls to take effect. 
+If your project is a [Direct Upload](/pages/get-started/direct-upload/) project, you will not have the option to configure production branch controls. To update your production branch, you will need to manually call the [Update Project](/api/operations/pages-project-update-project) endpoint in the API.
+
+```bash
+curl -X PATCH "https://api.cloudflare.com/client/v4/accounts/<ACCOUNT_ID>/pages/projects/<PROJECT_NAME>" \
```
Comment (deadlypants1973) [inline content/pages/configuration/branch-build-controls.md]:
```suggestion
curl -X PATCH "https://api.cloudflare.com/client/v4/accounts/{account_id}/pages/projects/{project_name}" \
```
for style guide, we decided to keep the `{}` formatting as the API docs do in regular docs.

**Comment by deadlypants1973** [comment]
> Looks like the same goal as the warning i added in #13180 - was this a better place to put it or is there value in having both?
> 
> The placement of mine was as recommended in [#13180 (review)](https://github.com/cloudflare/cloudflare-docs/pull/13180#pullrequestreview-1923117794)
> 
> If we do need both, Kate's feedback on the curl placeholders would apply here too: https://github.com/cloudflare/cloudflare-docs/pull/13180/files/43831d91616de4d9b2a7173384f450792ae85088#diff-261851dd6ea58e2a549870365f622562ae5e64b66f57cbb3d1cc567423971469

Is this saying the same thing?

> Looks like the same goal as the warning i added in #13180 - was this a better place to put it or is there value in having both?
> 
> The placement of mine was as recommended in [#13180 (review)](https://github.com/cloudflare/cloudflare-docs/pull/13180#pullrequestreview-1923117794)
> 
> If we do need both, Kate's feedback on the curl placeholders would apply here too: https://github.com/cloudflare/cloudflare-docs/pull/13180/files/43831d91616de4d9b2a7173384f450792ae85088#diff-261851dd6ea58e2a549870365f622562ae5e64b66f57cbb3d1cc567423971469

These look like we are saying the same thing? Can we make one partial and add them in all relevant places? @WalshyDev

**Pull request merged**